lib/const.rb

```diff
@@ -307,25 +307,25 @@
 
   Usage:
     crew build [options] [-k|--keep] [-v|--verbose] <name> ...
-    crew check [-V|--version] [-v|--verbose] <name> ...
-    crew const [-v|--verbose] [<name> ...]
+    crew check [options] [-V|--version] [-v|--verbose] <name> ...
+    crew const [options] [-v|--verbose] [<name> ...]
     crew deps [options] [--deep] [-t|--tree] [-b|--include-build-deps] [--exclude-buildessential] [-v|--verbose] <name> ...
     crew download [options] [-s|--source] [-v|--verbose] <name> ...
-    crew files <name> ...
+    crew files [options] <name> ...
     crew help [<command>] [-v|--verbose] [<subcommand>]
     crew install [options] [-k|--keep] [-s|--source] [-S|--recursive-build] [-v|--verbose] <name> ...
     crew list [options] [-v|--verbose] (available|installed|compatible|incompatible)
     crew postinstall [options] [-v|--verbose] <name> ...
-    crew prop [<property>]
+    crew prop [options] [<property>]
     crew reinstall [options] [-k|--keep] [-s|--source] [-S|--recursive-build] [-v|--verbose] <name> ...
-    crew remove [-v|--verbose] <name> ...
-    crew search [-v|--verbose] <name> ...
-    crew sysinfo [-v|--verbose]
-    crew test [-v|--verbose] [<name> ...]
+    crew remove [options] [-v|--verbose] <name> ...
+    crew search [options] [-v|--verbose] <name> ...
+    crew sysinfo [options] [-v|--verbose]
```

Comment:
Why are we adding all these back? For example, `crew sysinfo` certainly does not take any other options than the ones already listed.

Reply:
docopt is broken without the [options] fields.
try this. It uses the `-d` flag to enable `--no-color`.
```
crew -d sysinfo
```
Now try this:
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=slow_remove crew update ; crew -d sysinfo
```
Only the latter usage works. docopt has been erroring out with these flags for a while... It's an old regression!

Reply:
That's because we didn't list `[-d |--no-color]` in crew sysinfo-- adding `[options]` will make docopt automatically accept all the listed options we have i.e. passing `--recursive-build` to `crew sysinfo` will be accepted. 

also-- `crew sysinfo` doesn't print any color, so it shouldn't need to accept the `--no-color` option. 

if there are some commands which need to support specific arguments which they don't currently, then by all means add those arguments to those commands, but adding `[options]` makes every command accept every argument, which is misleading.

Reply:
Feel free to submit a PR or add a change to this PR to handle that. Just make sure `crew -d <command>` doesn't break things.

Reply:
To add further impetus... NOT having `[options]` there for everything BREAKS Docopt... so we need to find a way around that, otherwise stuff like `--no-color` causes crew to error out in docopt before args get passed to `crew` from it.